src/test/java/com/microsoft/azure/vmagent/test/IntegrationTest.java

```diff
@@ -135,7 +132,7 @@ public static JenkinsRule jenkinsRule() {
             clientId = TestEnvironment.loadFromEnv("VM_AGENTS_TEST_CLIENT_ID");
             clientSecret = TestEnvironment.loadFromEnv("VM_AGENTS_TEST_CLIENT_SECRET");
 
-            oauth2TokenEndpoint = TestEnvironment.loadFromEnv("VM_AGENTS_TEST_TOKEN_ENDPOINT");
+            tenantId = TestEnvironment.loadFromEnv("VM_AGENTS_TEST_TENANT_ID");
```

Comment:
The token endpoint is deprecated and there was no docs on what the value should be, the code is just parsing the tenant id out of it.
May need a CI update but I think its worth doing this?

Reply:
I will check and fix this.